src/native.js

```diff
@@ -229,9 +345,24 @@ export function getAllAssetsMessage(data, adObject) {
   const message = {
     message: 'assetResponse',
     adId: data.adId,
-    assets: []
   };
 
+  // Pass to Prebid Universal Creative all assets, the legacy ones + the ortb ones (under ortb property)
+  const ortbRequest = nativeMapper.get(adObject.requestId);
```

Comment:
I'm confused by this, could you add a comment outlining the message format and how you are updating it?  why does this need to update the bid response (`adObject`) ? If that is necessary I don't think it should be done here.

Reply:
adObject is a bid response; what doesn't seem right is that you need to request a message to update the bid response. 
If it's correct to do the last step (`bidResponse.native = ....`), then it should be done around the time it's returned by the adapter. If what matters is the message, you should be able to populate `message.assets` & `message.ortb` without modifying the bid response.

Which one it is depends on whether the legacy format matters anywhere but these messages.